resources/quality/tinyboy/tinyboy_fabrikator15_draft.inst.cfg

```diff
@@ -31,7 +31,7 @@ layer_height_0 = 0.3
 material_bed_temperature = 60
 material_diameter = 1.75
 material_print_temperature = 200
-material_print_temperature_layer_0 = 0
+material_print_temperature_layer_0 = 200
```

Comment:
If you just _remove_ the `material_print_temperature_layer_0` line, Cura will _automatically_ inherit the value from `material_print_temperature` (which I think you are after.